### DISCUSSION TITLE
Compile error

### DISCUSSION THREAD

**Discussion by k-ilias** [akvorado/akvorado#25] [Q&A] [answered]
Hello, i see akvorado in Telegram and want to test it. But ia have error with compile on debian 11.
Please tell me what is wrong?
![akvorado compile](https://user-images.githubusercontent.com/44038537/180225483-2b76e83f-a4e3-48b4-bac3-8894105efc73.png)

**Comment by vincentbernat** [comment]
I think you have some outdated node version lying around. Check `node --version` and `nodejs --version`. Also, maybe you don't need to compile it. The [release page](https://github.com/vincentbernat/akvorado/releases/tag/v1.5.0) contains a precompiled version.

**Reply by k-ilias** [reply]
Thanks, you are right

**Comment by charlyforot** [comment, marked as answer]
Hello,

I had the same problem on Debian using apt.
If you run `apt install nodejs`, it will install nodeJS v12 by default.
This version doesn't seem to work with akvorado source code.

To install nodeJS v16 on Debian:

```
sudo apt remove nodejs
curl -sL https://deb.nodesource.com/setup_16.x | sudo bash -
sudo apt install -y nodejs
```

To check:
```
node --version
v16.16.0
```

**Reply by vincentbernat** [reply]
Yes, at least Vite.js (the builder) requires Node 14.18+ or 16+. I'll update the doc to tell we need Node 16+.
